codeforamerica/vita-min
# MD 502: Additions section tax calculator, XML elements and PDF fields

## Link to pivotal/JIRA issue
- e.g.https://codeforamerica.atlassian.net/browse/FYST-896?atlOrigin=eyJpIjoiZDIyOGQ0YjRlMjI0NGJmY2JlMzI2ZmY1ZjgwZjkxMTYiLCJwIjoiaiJ9
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Adding tax calculations, XML and pdf filler for MD 502 Additions, line 3, 6 and 7
## How to test?
- Create a intake with `box14_stpickup` values on their w-2 to test line 3
- check that line 6 is equal to line 3
- check that line 7 is equal to line 6 plus federal AGI
## Screenshots (for visual changes)
<img width="564" alt="Screenshot 2024-11-06 at 4 39 15 PM" src="https://github.com/user-attachments/assets/1b2e7ff7-f568-4040-810f-883ac6b552c2">
<img width="647" alt="Screenshot 2024-11-06 at 4 40 36 PM" src="https://github.com/user-attachments/assets/57cdd654-ad5a-4684-9f5a-767c76223199">
